src/components/Login.js

```diff
@@ -1,10 +1,14 @@
 import React, {useState} from 'react';
+import "../styles/Form.css"
```

Comment:
Can you add a semi-colon at the end of the line so that its formatted correctly?